packages/lit-dev-content/src/global/dsd-polyfill.ts

```diff
@@ -0,0 +1,5 @@
+import {hydrateShadowRoots} from '@webcomponents/template-shadowroot/template-shadowroot.js';
+
+// Polyfill will short if the browser supports native DSD
```

Comment:
True, though it does it in a more costly way than necessary I think (https://github.com/webcomponents/template-shadowroot/blob/master/src/_implementation/feature_detect.ts could just be `HTMLTemplateElement.prototype.hasOwnProperty('shadowRoot')`). We should probably just update the polyfill, though, so no action needed.